tests/integration/integration/home/home-en.spec.js

```diff
@@ -0,0 +1,23 @@
+import { Selector } from "testcafe";
+import { googleAuthUser } from "../../helpers/loginByGoogle";
+import { selectLocaleFromDialog } from "../../helpers/selectLocale";
+
+fixture("Home - English")
+  .page(`${process.env.BROWSERSTACK_BASE_URL}/login`)
+  .beforeEach(async (t) => {
+    await t
+      .useRole(googleAuthUser)
+      .navigateTo(`${process.env.BROWSERSTACK_BASE_URL}/home`);
```

Comment:
ideally, if we could use clicks on the interface for `googleAuthUser`, we wouldn't have to add a separate line for doing this, right? it would happen automatically?

Reply:
See my comment here: https://github.com/avantifellows/plio-frontend/pull/568#discussion_r790790648